docker/Dockerfile

```diff
@@ -235,9 +229,7 @@ FROM fv3gfs-build AS fv3gfs-compiled
 
 RUN apt-get update && apt-get install -y \
     python3 \
-    python3-pip && \
-    ln -s /bin/python3 /bin/python && \
-    ln -s /bin/pip3 /bin/pip
```

Comment:
Without removing this line creating the symlink for `pip3` fails since `pip` already exists in `/bin`:
```
ln: failed to create symbolic link '/bin/pip': File exists
```
I don't believe these symlinks are needed in general, so I removed the one above as well.